pylot/flags.py

```diff
@@ -1,9 +1,5 @@
 from absl import flags
 
-import pylot.control.flags
```

Comment:
Can you please keep these imports? I think there's some pylot or data_gathering configs that do not work if we don't explicitly include the flags here. I'm more than happy to remove the imports, but we have to check that all configs.

Reply:
Sounds good, sorry about that